src/makeCalculator.js

```diff
@@ -37,7 +37,34 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    add(num) {
+      calculator.result += num;
```

Comment:
use `this`. apply for other operations as well

```suggestion
      this.result += num;
      return this;
```